src/HtmlReader/index.tsx

```diff
@@ -100,6 +100,13 @@ export default function useHtmlReader(args: ReaderArguments): ReaderReturn {
         navHeight: HEADER_HEIGHT,
         margin: 0,
       },
+      rights: {
+        /**
+         * Makes the reader fetch every resource before rendering, which
+         * takes forever.
+         */
+        autoGeneratePositions: false,
```

Comment:
Are you setting this to false so that the reader doesn't do this, or is the reader currently doing this for some other reason?

Reply:
This setting got defaulted to `true` in a recent update to R2D2BC. I don't think it should be that way because it makes the reader fetch every resource before rendering. I am here manually turning it off. We don't need it anyway for the features we are using.